docs/docs/genai/tracing/quickstart/index.mdx

```diff
@@ -0,0 +1,134 @@
+import ImageBox from '@site/src/components/ImageBox';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsWrapper from '@site/src/components/TabsWrapper';
+
+# Tracing Quickstart
+
+This quickstart guide will walk you through setting up a simple GenAI application with MLflow Tracing. In less than 10 minutes, you'll enable tracing, run a basic application, and explore the generated traces in the MLflow UI.
+
+## Prerequisites
+
+Make sure you have started the MLflow server.
```

Comment:
Could we add a command to run the server directly? As a user, I wouldn’t want to navigate to another page just to figure out how to start the server.